docker-compose.dev.yml

```diff
@@ -55,81 +87,60 @@ services:
     env_file: .env
     environment:
       - NEXT_PUBLIC_ENV=development
-      - INFISICAL_TELEMETRY_ENABLED=${TELEMETRY_ENABLED}
-    networks:
-      - infisical-dev
-
-  mongo:
-    image: mongo
-    container_name: infisical-dev-mongo
-    restart: always
-    env_file: .env
-    environment:
-      - MONGO_INITDB_ROOT_USERNAME=root
-      - MONGO_INITDB_ROOT_PASSWORD=example
-    volumes:
-      - mongo-data:/data/db
-    networks:
-      - infisical-dev
+      - INFISICAL_TELEMETRY_ENABLED=false
 
-  mongo-express:
-    container_name: infisical-dev-mongo-express
-    image: mongo-express
+  pgadmin:
+    image: dpage/pgadmin4
     restart: always
-    depends_on:
-      - mongo
-    env_file: .env
     environment:
-      - ME_CONFIG_MONGODB_ADMINUSERNAME=root
-      - ME_CONFIG_MONGODB_ADMINPASSWORD=example
-      - ME_CONFIG_MONGODB_URL=mongodb://root:example@mongo:27017/
+      PGADMIN_DEFAULT_EMAIL: admin@example.com
+      PGADMIN_DEFAULT_PASSWORD: pass
     ports:
-      - 8081:8081
-    networks:
-      - infisical-dev
+      - 5050:80
+    depends_on:
+      - db
 
   smtp-server:
     container_name: infisical-dev-smtp-server
     image: lytrax/mailhog:latest # https://github.com/mailhog/MailHog/issues/353#issuecomment-821137362
     restart: always
     logging:
-      driver: 'none' # disable saving logs
+      driver: "none" # disable saving logs
     ports:
       - 1025:1025 # SMTP server
       - 8025:8025 # Web UI
-    networks:
-      - infisical-dev
 
-  redis:
-    image: redis
-    container_name: infisical-dev-redis
-    environment:
-      - ALLOW_EMPTY_PASSWORD=yes
-    ports:
-      - 6379:6379
-    volumes:
-        - redis_data:/data
-    networks:
-      - infisical-dev
-
-  redis-commander:
-    container_name: infisical-dev-redis-commander
-    image: rediscommander/redis-commander
-    restart: always
-    depends_on:
-      - redis
-    environment:
-      - REDIS_HOSTS=local:redis:6379
-    ports:
-      - "8085:8081"
-    networks:
-      - infisical-dev
+  # mongo:
```

Comment:
Since we don't need Mongo, let's delete these

Reply:
Oke